runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -269,7 +269,7 @@ public void onConfigChanged(ConfigChanged event)
 	protected void startUp() throws Exception
 	{
 		ignoredItems = Text.fromCSV(config.getIgnoredItems());
-		panel = new LootTrackerPanel(this, itemManager, config);
+		panel = new LootTrackerPanel(this, itemManager, config, injector.getInstance(ScheduledExecutorService.class));
```

Comment:
We already have access to a SES instance in the injected `executor` in this plugin. Why isn't that used?

Reply:
I suppose I assumed there wasn't one, thanks for the remark.